logger/simple.go

```diff
@@ -5,10 +5,12 @@ import (
 	"log"
 )
 
-type SimpleLogger struct{}
+type SimpleLogger struct {
+	Level Level
+}
 
 func (l *SimpleLogger) Enabled(_ context.Context, level Level) bool {
-	return true
+	return level >= l.Level
 }
```

Comment:
please add `Enabled(..)` calls to every method in the simple logger. otherwise, this has no effect :) 

e.g. if we call `logger.Debugf(..)` and we have `logging.LevelInfo` set, it shouldn't print anything

Reply:
Please check 👍